packages/core/src/utils/Utils.ts

```diff
@@ -514,6 +515,35 @@ export class Utils {
       || !!new Error().stack!.split('\n').find(line => line.match(/\w\.ts:\d/)); // as a last resort, try to find a TS file in the stack trace
   }
 
+  private static async getJestConfig(): Promise<Dictionary> {
+    if (await pathExists(process.cwd() + '/jest.config.json')) {
+      return require(process.cwd() + '/jest.config.json');
+    }
+    return {};
+  }
+
+  private static jestConfigUsesTsJest(jestConfig: Dictionary): boolean {
+    return jestConfig.preset === 'ts-jest'
+      || (Object.values(jestConfig.transform ?? {}).some(transformer => transformer === 'ts-jest'));
```

Comment:
not a big fan of splitting stuff to multiple lines randomly (prettier style, meh). much better to create new local variable (human style, lol):

```suggestion
    const usesJestTransformer = Object.values(jestConfig.transform ?? {}).some(transformer => transformer === 'ts-jest');
    return jestConfig.preset === 'ts-jest' || usesJestTransformer;
```